pyproject.toml

```diff
@@ -109,9 +108,27 @@ ignore_missing_imports = true
 module = [
     "exasol/nb_connector/extension_wrapper_common",
     "test.*",
-    ]
+    # not yet include py.typed
+    "docker.*",
+    "ibis.*",
+    "pyexasol.*",
+    # not supporting mypy > 1.10.0; https://docs.sqlalchemy.org/en/20/orm/extensions/mypy.html
+    "sqlalchemy.*",
+    # typed from specific version
+    "exasol.bucketfs.*", # >=1.0.1
+    "exasol_integration_test_docker_environment.*", # >=3.3.0
+    "exasol.saas.*", # >=2.2.0
+    "exasol.slc.*", # >= 1.0.0
```

Comment:
Why do we need those packages in the list? I believe most of them have proper type hints....

Reply:
Ah, yes, this list was defensively created assuming the at #207 wasn't merged & that there were maybe some type issues I wasn't aware of (can't remember the thread/context but thought you had mentioned it in the last daily wrt NC, but maybe it was that a specific type hint had wandered.)

All the ones listed here currently have # type: ignore's throughout the code (which I can add a commit removing those one-line and reducing this list too to what's needed based on our current pinned python packages).